publication-doi-commons/src/main/java/no/unit/nva/publication/doi/dto/Contributor.java

```diff
@@ -0,0 +1,114 @@
+package no.unit.nva.publication.doi.dto;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.net.URI;
+import java.util.Objects;
+import no.unit.nva.publication.doi.dynamodb.dao.Identity;
+import nva.commons.utils.JacocoGenerated;
+
+public class Contributor {
+
+    private final URI id;
+    private final String arpId;
+    private final String name;
+
+    /**
+     * Constructs a Contributor for doi.publisher DTO
+     * @param id URI for our contributor. Null atm.
+     * @param arpId Authority register id for a person for this contributor entry.
+     * @param name name of contributor
+     */
+    @JacocoGenerated
+    @JsonCreator
+    public Contributor(@JsonProperty("id") URI id,
+                       @JsonProperty("arpId") String arpId,
+                       @JsonProperty("name") String name) {
+        this.id = id;
+        this.arpId = arpId;
+        this.name = name;
+    }
+
+    /**
+     * Constructs a Contributor for doi.publisher DTO.
+     *
+     * @param identity Identity DAO from Dynamodb representing a identity contributor.
+     */
+    @SuppressWarnings("PMD.NullAssignment")
+    public Contributor(Identity identity) {
+        this.id = null;
```

Comment:
Since we now have a `ContributeMapper` (we didnt before), im dropping the constructor for it having to eat a DAO, and rather advise to use the `ContributeMapper`.